build-system/tasks/check-analytics-vendors-list.js

```diff
@@ -71,7 +71,10 @@ async function checkAnalyticsVendorsList() {
     const name = nameMatch
       .split(/[,\s]+/)
       .shift()
-      .replace(/[`"']/g, '');
+      ?.replace(/[`"']/g, '');
```

Comment:
Use `!` and let it fail fast.

Reply:
Can't! This isn't a JS file

Reply:
Arg. Keeping it as-is, lmk if you'd like a followup